### ISSUE TITLE
[Main menu site admin] Org admin menu is hidden but still part of the DOM

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#6411]
## Area or Component

Main menu as site admin

## Quick Summary

The menu list for the organization admin is still part of the DOM even though it is "hidden" for the sighted users.

### How to Replicate

Check the DOM.

## Accessibility Violation
[Content is not a list but it is marked as such.](https://docs.deque.com/issue-help/1.0.0/en/semantic-list-used-for-presentation)

## Action Requested

Add logic to not include the `ul` menu element in the DOM for site admins.

## Screenshots (optional)
See how the DOM is still part of the page but is not visible because the menu items don't have text.

<img width="1420" alt="Screenshot 2023-08-24 at 10 41 49 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/23abffc9-a2f7-468b-b4ee-b51ee73caf1e">
